src/app/components/login/login.component.ts

```diff
@@ -20,16 +21,16 @@ export class LoginComponent implements OnInit {
   }
 
   submit() {
+
       let formData: FormData = new FormData();
-      formData.append('_username', this.username)
-      formData.append('_password', this.password)
+      formData.append('_username', this.username);
+      formData.append('_password', this.password);
 
-      var headers = new Headers();
+      let headers = new Headers();
       headers.append('Content-Type', 'application/x-www-form-urlencoded');
```

Comment:
headers is no longer requirered , you can remove those two lines

Reply:
Ok, i'll edit this when i back at home